index.html

```diff
@@ -65,38 +67,104 @@ <h2 class="my-name">I'm Yemsrach Glad to see you!</h2>
     </div>
     </div>
     </section>
-  
-    <section class="work">
-       <section class="work">
+    <!-- Home section start -->
+  <!-- wotk section start -->
+  <section class="work grid-container">
 <div class="snapshoot">
   <img src="./image/Snapshoot Portfolio.png" alt="snapchat">
-  <div class="snapshoot_1"></div>
-  <h1>Tonic</h1>
+  <div class="snapshoot_1">
+  <h2>Tonic</h2>
   <ul class="profile">
   <li>CANOPY</li>
   <li><img src="./image/Ellipse 5.png" alt="ellipse"></li>
   <li>Back End Dev</li>
   <li><img src="./image/Ellipse 5.png" alt="ellipse"></li>
   <li>2015</li>
   </ul>
+</div>
   <p>A daily selection of privately presonalizaed reads: no accounts or singn-up required.</p>
   <ul class="skill">
-    <li><h4>html</h4></li>
-  <li>css</li>
-  <li>javaScript</li>
+    <li><button>html</button></li>
+  <li><button>css</button></li>
+  <li class="javaScript"><button>javaScript</button></li>
   </ul>
-  <button class="btn"><a href="#">See Project</a></button>
+  <a class="see_project" href="#"><button>See Project</button></a>
```

Comment:
As requested by the previous reviewer, kindly check your card container,(work-grid container) it is not taking the full width of the mother container 👇
<img width="427" alt="image" src="https://user-images.githubusercontent.com/73854999/205436690-70ffc2c6-d08d-4edb-a729-7979b98af23e.png">

Reply:
Fixed.